artifactory/groups.go

```diff
@@ -54,11 +55,23 @@ func (s *GroupsService) GetAll() (*[]Group, *Response, error) {
 	return v, resp, err
 }
 
+// GetGroupRequest represents the possible inputs for a Get Group request.
+//
+// Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
+type GetGroupRequest struct {
+	Name         *string
+	IncludeUsers *bool
+}
+
 // Get returns the provided group.
 //
 // Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
-func (s *GroupsService) Get(group string) (*Group, *Response, error) {
-	u := fmt.Sprintf("/api/security/groups/%s", group)
+func (s *GroupsService) Get(groupRequest *GetGroupRequest) (*Group, *Response, error) {
+	u := fmt.Sprintf("/api/security/groups/%s", *groupRequest.Name)
+	if groupRequest.IncludeUsers != nil {
+		u = u + fmt.Sprintf("?includeUsers=%t", *groupRequest.IncludeUsers)
```

Comment:
Similar to above, thoughts on potentially using the struct accessor instead?

i.e.

```go
u = u + fmt.Sprintf("?includeUsers=%t", groupRequest.GetIncludeUsers())
```

Reply:
I think @JordanSussman's [suggestion](https://github.com/target/go-arty/pull/56#discussion_r431240256) would work well here instead, particularly for its backwards compatibility.

Reply:
@joeh90 sounds good!